templates/index.html

```diff
@@ -177,6 +177,8 @@ <h4 class="text-center">
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
 <script type="text/javascript" src="{{ url_for('static', filename='js/jMinEmoji2.min.js') }}"></script>
 <script type="text/javascript" src="{{ url_for('static', filename='js/script.js') }}"></script>
+<script type="text/javascript" src="{{ url_for('static', filename='js/linkifyjs/linkify.min.js') }}"></script>
+<script type="text/javascript" src="{{ url_for('static', filename='js/linkifyjs/linkify-jquery.min.js') }}"></script>
```

Comment:
There is already `jquery 3.4.1` present, can you please verify if this is necessary or not?
We should only use one jquery.

Reply:
Not 100% sure, but a test with out the second (new) jquery works fine. So I think we coudl remove this
```suggestion

```